CalibTracker/SiStripESProducers/plugins/services/modules.cc

```diff
@@ -40,3 +40,7 @@ DEFINE_FWK_SERVICE(SiStripBaseDelayGenerator);
 #include "FWCore/ServiceRegistry/interface/ServiceMaker.h"
 #include "CalibTracker/SiStripESProducers/interface/SiStripConfObjectGenerator.h"
 DEFINE_FWK_SERVICE(SiStripConfObjectGenerator);
+
+#include "FWCore/ServiceRegistry/interface/ServiceMaker.h"
+#include "CalibTracker/SiStripESProducers/interface/SiStripBadModuleFedErrService.h"
+DEFINE_FWK_SERVICE(SiStripBadModuleFedErrService);
```

Comment:
@davidlange6 @suchandradutta This piece of code has been introduced by Suchandra, I let her to comment. Anyway I am closing this PR since there is a new one from Suchandra that overseeds this https://github.com/cms-sw/cmssw/pull/13888